cscan.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
 pushd "$(dirname ${BASH_SOURCE[0]})" > /dev/null
 if [ ! -d ./tlslite ]; then
-    git clone --depth=1 https://github.com/tomato42/tlslite-ng.git .tlslite-ng
+    git clone --depth=1 https://github.com/tomato42/tlslite-ng.git .tlslite-ng &>/dev/null
```

Comment:
good point on the "I got no idea where that came from"

how about prepending "cscan.sh: " to the `not found` messages?